src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -174,7 +174,7 @@ export class AudioCueService extends Disposable implements IAudioCueService {
 		() => /** @description config: audioCues.enabled */ this.configurationService.getValue<'on' | 'off' | 'auto' | 'userGesture' | 'always' | 'never'>('audioCues.enabled')
 	);
 
-	private readonly isCueEnabledCache = new Cache((event: IAudioCueEvent) => {
+	private readonly isCueEnabledCache = new Cache((event: { readonly cue: AudioCue; readonly userGesture?: boolean }) => {
```

Comment:
Now, you could shorten the type to
```ts
{ cue: AudioCue; userGesture?: boolean }
```
or even use a different type (and adjust callers to `getValue`):
```ts
[AudioCue, userGesture?: boolean]
```
... but that is just icing on the cake.

Reply:
I tried this to no avail 🤔